compiler/hash-ast/src/resolve.rs

```diff
@@ -118,15 +120,14 @@ where
                     },
                     Level::Debug,
                     |elapsed| println!("ast: {:.2?}", elapsed),
-                )?;
+                );
+
+                // @@Hack: we still need to add the contents of the file into the map
```

Comment:
Why is it necessary to split contents from node?

Reply:
Yeah this was just a silly workaround whilst hooking up the reporting logic.